bsp/avr32uc3b0/application.c

```diff
@@ -37,12 +37,12 @@ void rt_thread_entry_led2(void* parameter)
     while (1)
     {
         /* led on */
-        rt_kprintf("LED2 on");
+        //rt_kprintf("LED2 on");
```

Comment:
remove it or release it. Please don't "//" it .

Reply:
Dear @supperthomas, I have removed the commented rt_kprintf statements. Thank you.